src/polygons/polygons.jl

```diff
@@ -68,25 +69,47 @@ function change_inclusion!(inclusion, layer, polygon, op)
     return inclusion
 end
 
+function _get_inclusion_from_polygon!(inclusion, layer, multipolygon::GeoJSON.MultiPolygon)
+    for element in multipolygon
+        for i in eachindex(element)
+            change_inclusion!(inclusion, layer, element[i], true)
+        end
+    end 
+end 
+
+function SimpleSDMLayers.mask!(layers::Vector{SDMLayer}, multipolygon::GeoJSON.MultiPolygon) 
+    inclusion = zeros(eltype(first(layers).indices), size(first(layers)))
```

Comment:
I think it's probably safer to do something like a mask from all layers with something like `reduce(.|, [l.grid for l in layers])` and then do the inclusion on this?

Reply:
agreed, updated